src/platform/user/authentication/selectors.js

```diff
@@ -33,3 +33,6 @@ export const isAuthenticatedWithSSOe = state =>
 
 export const ssoeTransactionId = state =>
   selectProfile(state)?.session?.transactionid;
+
+export const transitionMHVAccount = state =>
+  state.user.profile.mhvTransitionEligible ?? false;
```

Comment:
should we add optional chaining here or will the `?? false` handle the short circuit if we run into a `cannot read x of undefined`?

Reply:
The nullish coalescing will work